lib/auth/clt.go

```diff
@@ -1465,12 +1468,79 @@ func (c *Client) SearchEvents(fromUTC, toUTC time.Time, namespace string, eventT
 func (c *Client) SearchSessionEvents(fromUTC time.Time, toUTC time.Time, limit int, startKey string) ([]events.AuditEvent, string, error) {
 	events, lastKey, err := c.APIClient.SearchSessionEvents(context.TODO(), fromUTC, toUTC, limit, startKey)
 	if err != nil {
-		return nil, "", trace.Wrap(err)
+		log.WithError(err).Debug(
+			`Attempted to call SearchSessionEvents over gRPC but received error,
+falling back to legacy API`)
+		return c.searchSessionEventsFallback(context.TODO(), fromUTC, toUTC, limit, startKey)
 	}
 
 	return events, lastKey, nil
 }
 
+// searchEventsFallback is a fallback version of SearchEvents that queries the deprecated
+// HTTP API if the auth server is on a version < 6.2.
+func (c *Client) searchEventsFallback(ctx context.Context, fromUTC, toUTC time.Time, namespace string, eventTypes []string, limit int, startKey string) ([]events.AuditEvent, string, error) {
```

Comment:
This should probably err if `startKey != ""`.  Its unlikely that this would happen, since it would mean that the first call had hit an auth server which supported the new GRPC api, and the second call hit one that didn't, but its worth making sure that we err on that case instead of returning a confusing/invalid event set.